picoquic/packet.c

```diff
@@ -1139,6 +1142,13 @@ int picoquic_incoming_encrypted(
         /* Packet is correct */
         if (ph->pn64 > cnx->first_sack_item.end_of_sack_range) {
             cnx->current_spin = ph->spin ^ cnx->client_mode;
+	    if (ph->hasspinbit && cnx->current_spin != cnx->prev_spin) {
+	      // got an edge 
+	      cnx->prev_spin = cnx->current_spin;
+	      cnx->Edge = 1;
+	      cnx->VEC = (ph->VEC==3) ? 3 : (ph->VEC + 1);
+	      cnx->lastTrigger =  picoquic_get_quic_time(cnx->quic);
+	    }
```

Comment:
Can you find some more expressive variable names than "Edge" or "VEC" or "LastTrigger"? Maybe something like "spin_edge", "spin_vec" and "spin_last_trigger"? My concern is that when someone later reads the code they do not understand the purpose of the variables.